src/services/add-service.js

```diff
@@ -1,6 +1,7 @@
 import { getAvailableComponents, findParentUriAndList } from './utils';
 import { findSpaceLogic } from './create-service';
-import { setNewActive } from '../services/toggle-service';
+import { setNewActiveLogic } from '../services/toggle-service';
+import { last } from 'lodash';
```

Comment:
I don't think this will actually tree-shake lodash when all of clay-space-edit is compiled, will it?

Reply:
ahh I think you're right, webpack is not doing any tree-shaking. Will investigate

Reply:
K, added Lodash tree-shaking